packages/synapse-react-client/src/components/AccessRequirementList/AccessRequirementList.tsx

```diff
@@ -358,39 +346,39 @@ export default function AccessRequirementList(
         )
         break
       case RequestDataStep.PROMPT_LOGIN:
+        dialogTitle = 'Please Log In'
         renderContent = (
-          <>
-            <DialogTitle>Please Log In</DialogTitle>
-            <DialogContent className={'AccessRequirementList login-modal '}>
-              <StandaloneLoginForm
-                sessionCallback={() => {
-                  window.location.reload()
-                }}
-              />
-            </DialogContent>
-          </>
+          <Box className={'AccessRequirementList login-modal '}>
+            <StandaloneLoginForm
+              sessionCallback={() => {
+                window.location.reload()
+              }}
+            />
+          </Box>
         )
         break
       case RequestDataStep.COMPLETE:
         renderContent = <RequestDataAccessSuccess onHide={onHide} />
         break
       case RequestDataStep.SHOW_ALL_ARS:
       default:
-        renderContent = (
-          <>
-            {content}
-            <DialogActions>
-              <Button variant="contained" onClick={onHide}>
-                Close
-              </Button>
-            </DialogActions>
-          </>
+        renderContent = content
+        dialogActions = (
+          <Button variant="contained" onClick={onHide}>
+            Close
+          </Button>
         )
     }
     return (
-      <Dialog maxWidth={dialogWidth} fullWidth open={true} onClose={onHide}>
-        {renderContent}
-      </Dialog>
+      <DialogBase
```

Comment:
In the `renderAsModal` case, use `DialogBase`, which makes this more consistent with our other dialog components.